sys/drv/ns16550.c

```diff
@@ -183,11 +186,12 @@ static int ns16550_attach(device_t *dev) {
   return 0;
 }
 
-static driver_t ns16550_driver = {
-  .desc = "NS16550 UART driver",
-  .size = sizeof(ns16550_state_t),
-  .attach = ns16550_attach,
-  .identify = bus_generic_identify,
-};
+DEVICE_DRIVER_GEN_PCI_PROBE(NS16550_VENDOR_ID, NS16550_DEVICE_ID, ns16550)
+
+static driver_t ns16550_driver = {.desc = "NS16550 UART driver",
+                                  .size = sizeof(ns16550_state_t),
+                                  .attach = ns16550_attach,
+                                  .identify = bus_generic_identify,
+                                  .probe = dev_generic_probe_ns16550};
```

Comment:
Why did you change the formatting here?

Reply:
I misued make format, it's corrected now.